makecrx.sh

```diff
@@ -38,7 +38,10 @@ echo "Building chrome version" $VERSION
 [ -d pkg ] || mkdir -p pkg
 [ -e pkg/crx ] && rm -rf pkg/crx
 
-# Only generate the sqlite database if any rulesets have changed. Tried
+# Clean up obsolete ruleset databases, just in case they still exist.
+rm -f "$PWD/src/chrome/content/rules/default.rulesets" "$PWD/src/defaults/rulesets.sqlite"
```

Comment:
Just to be paranoid, I would leave out the -f (I don't think it's needed), and would use `.` instead of `$PWD`, just in case `$PWD` is not set for any weird reason, so it doesn't have an unexpected affect. Same below.